README.md

```diff
@@ -11,16 +11,17 @@ This is still a work in progress so some details might be changing. Feel free to
 This repository is split into the following modules:
 
     allshared
-    ├── analytics
-    ├── breeds
-        ├── database
+    analytics
```

Comment:
Can probably remove the list since it isn't showing a hierarchy anymore and they are each defined below